TRANSLATE.rst

```diff
@@ -0,0 +1,52 @@
+******************
+Translation Guide 
+******************
+
+.. contents:: 
+
+You can help to translate the Read the Docs Sphinx Theme.
+
+Languages availables:
+
+- Spanish contributed by Leonardo J. Caballero G.
+- Your Language by YOUR NAME.
```

Comment:
This file should be moved into our docs. Also, I'm very happy to apply attribution, but putting author and translation status information in the docs is guaranteed to go stale. I'd rather avoid this and keep authors in another location.

Reply:
@agjohnson @Blendify when you said "This file should be moved into our docs" are you refernce into [the docs readthedocs.org](https://github.com/rtfd/readthedocs.org/tree/master/docs)? Or what kind of location are you preference? or what other location are you preference?